node/src/initialise.rs

```diff
@@ -0,0 +1,30 @@
+//! Node initialisation routine.
+
+use crate::options::Options;
+use consensus::rpc::ConsensusRpcServer;
+
+pub async fn initialise(opts: Options) -> anyhow::Result<subsystem::Manager> {
+    // Initialise storage and chain configuration
+    let storage = blockchain_storage::Store::new_empty()?;
+    let chain_config = common::chain::config::create_mainnet();
```

Comment:
This should come from Options; we should be able to create any type of net we need: `--net=mainnet` (which is the default), `--net=testnet`, `--net=regtest`, etc. The `net` thing is up for discussion if anyone doesn't like it.

We should also support flags, like `--testnet` or `--regtest`, just to make it like bitcoin.

Reply:
Sure. At the moment, the chain config only exports `create_mainnet` so I did not bother with it for now.